nx-X11/programs/Xserver/hw/nxagent/NXcomposite.c

```diff
@@ -1,359 +0,0 @@
-#ifndef NXAGENT_UPGRADE
```

Comment:
Removing that file is a bad idea! It's not equivalent to `composite.h`! It's somewhat equivalent to `Xcomposite.h`, but has modifications by NoMachine as explained in my previous comment.

Please do not delete this file.

You may delete the `#ifndef NXAGENT_UPGRADE` guard, though, probably.